contracts/CourtRoom.sol

```diff
@@ -183,20 +218,41 @@ contract SwearGame is Owned {
 		registeredPlayersCounter--;
```

Comment:
@zelig see register function . 
It check if there is enough deposit before register a new client to the game.